Modules/Loadable/Segmentations/SubjectHierarchyPlugins/qSlicerSubjectHierarchySegmentationsPlugin.cxx

```diff
@@ -262,12 +268,24 @@ bool qSlicerSubjectHierarchySegmentationsPlugin::reparentItemInsideSubjectHierar
     return false;
     }
 
+  vtkSlicerSegmentationsModuleLogic* sgmentationsModuleLogic =
```

Comment:
It is simpler to get module logics via the application logic class - see here: https://github.com/Slicer/Slicer/blob/8694fc789671ef66644d10fff66d9c37826c40bd/Modules/Loadable/Markups/SubjectHierarchyPlugins/qSlicerSubjectHierarchyMarkupsPlugin.cxx#L291-L303